src/components/TimePicker/__test__/ampmSelect.spec.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import AmPmSelect from './../ampmSelect';
+
+describe('<AmPmSelect/>', () => {
+    it('should render the right markup initially when component is not focused', () => {
+        const component = mount(<AmPmSelect tabIndex="-1" />);
+        component.setState({ isFocused: false });
+        expect(
+            component.find('input[className="rainbow-time-picker_time-select-value"]').exists(),
+        ).toBe(true);
+    });
+    it('should render the right markup when component is focused', () => {
+        const component = mount(<AmPmSelect />);
+        component.setState({ isFocused: true });
+        expect(
+            component
+                .find(
+                    'fieldset[className="rainbow-time-picker_time-select-value rainbow-time-picker_select-ampm"]',
+                )
+                .exists(),
+        ).toBe(true);
+        expect(component.find('input[value="AM"]').exists()).toBe(true);
+        expect(component.find('input[value="PM"]').exists()).toBe(true);
+    });
+    it('should fire the onChange event with "AM" when the component get focus and value and defaultValue are undefined', () => {
+        const onChangeMckFn = jest.fn();
+        const component = mount(<AmPmSelect onChange={onChangeMckFn} />);
+        component.setState({ isFocused: false });
+        component
+            .find('input[className="rainbow-time-picker_time-select-value"]')
+            .simulate('focus');
+        expect(component.prop('onChange')).toHaveBeenCalledWith('AM');
+        expect(component.state('isFocused')).toBe(true);
```

Comment:
this are the stuff we should avoid to test, also in a previous test we already test this but checking what is important: the UI rendered